libs/SmartSync/src/com/salesforce/androidsdk/smartsync/util/SyncDownTarget.java

```diff
@@ -0,0 +1,133 @@
+/*
```

Comment:
Simply renamed SyncTarget to SyncDownTarget
But because I also introduced a interface SyncTarget (implemented by SyncDownTarget and SyncUpTarget), the diff doesn't see SyncDownTarget as a simple rename.